修改几个错别字

kernight opened the pull request:
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

hustnaive commented on docs/guide-zh-CN/db-query-builder.md:
```diff
@@ -0,0 +1,570 @@
+查询构建器
+=============
+
+查询构建器建立在 [Database Access Objects](db-dao.md) 基础之上。通过它可以创建一条程序化的并且DBMS无关的SQL语句。相比较于写原生的SQL语句，查询构建器可以帮你写出可读性更强的SQL相关的代码，并生成安全性更强的SQL语句。
```
通过它可以程序化的创建一条DBMS无关的SQL语句。

pkirill99 committed:
change echo to return

close #10771

samdark committed:
Merge branch 'master' into 2.1

samdark committed:
Remove deprecated methods and constants

samdark committed:
Changelog

samdark committed:
Use current cache method names in tests

samdark committed:
Merge branch 'master' into 2.1

samdark committed:
Fixed merge artifact in CHANGELOG

samdark committed:
Fixes #11397: Minimum required version of PHP is 5.6.0 now

cebe committed:
Use ::class instead of ::className()

- shorter syntax
- native PHP feature, so it does not depend on yii\base\Object
- Does not trigger autoloading anymore

cebe committed:
use ::class also in documentation and configs

- shorter syntax
- native PHP feature, so it does not depend on yii\base\Object
- Does not trigger autoloading anymore

cebe committed:
removed yii\base\Object::className()

it is easy to upgrade existing code and it serves no purpose since
::class syntax is available.

cebe committed:
::class syntax in the Guide

only partially. covered ::className() replacement and the
concept-configuration.md

cebe committed:
fixed tests for ::class syntax

required php version is 5.5 at least so 5.4 removed from travis

cebe committed:
more test fixes for ::className() -> ::class

cebe committed:
expression support in orderBy and groupBy (#10693)

this is the BC breaking solution for #10682

to be merged for 2.1.x

cebe committed:
UPGRADE note about #10693

Faryshta committed:
fix #11473 trailing backslash in class (#11474)

cebe committed:
fixed test break

wrong variable name

Faryshta committed:
simplify `::class` usage

Faryshta committed:
simplify default classes

Faryshta committed:
more changes

cebe committed:
Merge pull request #11477 from Faryshta/patch-2

Use PHP 5.5+ `::class` instead of full class namespace

samdark committed:
Merge branch 'master' into 2.1

# Conflicts:
#	framework/UPGRADE.md
#	framework/behaviors/SluggableBehavior.php

samdark committed:
Fixed usage of ::className() instead of ::class

lennartvdd committed:
Automatically turn off field label on ActiveField->hiddenInput()

Proposed solution for yiisoft/yii2#2990

lennartvdd committed:
Update CHANGELOG.md

samdark committed:
Merge branch 'master' into 2.1

samdark committed:
Removed deprecated method

cebe committed:
Consistent behavior of `run()` method in widgets

All widgets return the result now for better extensibility

close #10771

cebe committed:
fixed breadcrumb widget test

cebe committed:
Merge branch 'master' into 2.1

* master: (28 commits)
  test against php 7.1
  use the same db connection for reading and writing data
  added tests classes for db related tests
  refactored database tests
  improved docs about using CORS with REST controllers
  fixed scalar value for IN condition
  guide section about comparing dates
  added errorHandler, Request and Response getter
  fixed tutorial about using yii 1 with yii 2
  note about JOIN syntax
  added note about varying source languages
  Updated language packs zh-CN and ZH-TW
  Fixes #8644: Fixed trying to ENABLE/DISABLE TRIGGER ALL on a view in PostgreSQL
  Fixes #11549: Fixed `ArrayHelper::getValue()` to work properly with float keys
  Unit test for #11549
  Fixes #11739: Fixed `ArrayHelper::index()` losing precision for float keys
  Fix russian translation for Data widgets page (#11776) [skip ci]
  Fixes #11774: Fixed incorrect recusuive symlinks check in FileHelper
  Fixed non-translation block of text (#11765)
  Fixes #10422, #9574: New proposal fix for ColumnSchemaBuilder not null column
  ...

Conflicts:
	framework/web/User.php

samdark committed:
Merge branch 'master' into 2.1

cebe committed:
applied missing params for #10682 to mssql and oci query builder

fixes #11839

cebe committed:
better method contract for ActiveRelationTrait

cebe committed:
revert ActiveRelationTrait abstract methods

this does not seem to work the same way as interfaces:

```
PHP Fatal error:  Declaration of
yii\db\ActiveRelationTrait::andWhere($condition) must be compatible with
yii\db\Query::andWhere($condition, $params = Array)
```

samdark committed:
Merge branch 'master' into 2.1

# Conflicts:
#	framework/di/Instance.php
#	tests/framework/filters/AccessRuleTest.php

samdark committed:
Merge branch 'master' into 2.1

samdark committed:
#12074: Updated `yii\widgets\ActiveField::hint()` method signature to match `label()`

samdark committed:
Replaced ::className() with ::class

samdark committed:
Fixes #11560: Removed XCache and Zend data cache support as caching backends

samdark committed:
Merge branch 'master' into 2.1

# Conflicts:
#	framework/caching/XCache.php

brandonkelly committed:
::className() => ::class in the docs

SilverFire committed:
Merge pull request #12585 from brandonkelly/2.1-class

Changed `::className()` references to `::class` in the docs

dynasource committed:
#12592 enhance performance by ignoring rules that (in the end) are never used

SilverFire committed:
Merge pull request #12607 from dynasource/12592-improve-performance-accesscontrol

enhance performance by ignoring rules that are never used #12592

ivanovyordan committed:
Fixes #11058: Add `$checkAjax` parameter to method `yii\web\Controller::redirect()` which controls redirection in AJAX and PJAX requests

samdark committed:
Merge branch 'master' into 2.1

# Conflicts:
#	.travis.yml
#	docs/guide-es/concept-events.md
#	docs/guide-ja/concept-events.md
#	docs/guide-pt-BR/concept-events.md
#	docs/guide-ru/concept-events.md
#	docs/guide-zh-CN/concept-events.md
#	docs/guide/concept-events.md
#	framework/console/controllers/AssetController.php
#	framework/web/Application.php
#	framework/web/UrlManager.php
#	framework/web/User.php
#	framework/widgets/ActiveForm.php
#	tests/framework/base/EventTest.php

samdark committed:
Fixed className() usage

rob006 committed:
Enable `UrlNormalizer` in `UrlManger` by default

rob006 committed:
Update guide

rob006 committed:
Update upgrade note

SilverFire committed:
Merge pull request #12704 from rob006/enable-url-normalizer

Enable `UrlNormalizer` in `UrlManger` by default

mikehaertl committed:
Issue #12938 Add $params to View::renderDynamic() (#12955)

* Issue #12938 Add $params to View::renderDynamic()

* - add tests
- typehint $params

* add change to upgrade.md

SilverFire committed:
Merge branch 'master' into 2.1

 Conflicts:
	framework/UPGRADE.md
	framework/base/Security.php
	framework/caching/Cache.php
	framework/caching/XCache.php
	framework/db/QueryBuilder.php
	framework/db/Schema.php
	framework/db/mssql/QueryBuilder.php
	framework/validators/Validator.php
	framework/web/UrlManager.php
	tests/framework/mutex/FileMutexTest.php

SilverFire committed:
Fixed composer.json to build on PHP 5.5+

SilverFire committed:
Replaced all `::className()` calls to `::class`

SilverFire committed:
Replaced `::className()` in PHPDocs and some classes

SilverFire committed:
Fixed tests

SilverFire committed:
`yii\widgets\GridView::run() added missing return. Updated UPGRADE.md

SilverFire committed:
Merge branch 'activeFormHiddenField' of git://github.com/lennartvdd/yii2 into lennartvdd-activeFormHiddenField

stream into a topic branch.

SilverFire committed:
Merge branch 'lennartvdd-activeFormHiddenField' into 2.1

SilverFire committed:
Fixed CHANGELOG

boehsermoe committed:
Captcha:: have to start with /, otherwise it does not work in modules. (#13006)

* Captcha:: have to start with /, otherwise it does not work in modules.

cebe committed:
ensure fallback format is consistent with ICU (#7800)

fixes #7770

SilverFire committed:
Fixed code style for tests

SilverFire committed:
Merge branch 'master' into 2.1

 Conflicts:
	docs/guide-fr/concept-events.md
	docs/guide-ja/concept-events.md
	docs/guide-ru/concept-events.md
	docs/guide/concept-events.md
	framework/caching/Cache.php
	framework/db/ActiveRelationTrait.php
	framework/db/mssql/QueryBuilder.php
	framework/web/User.php

SilverFire committed:
Fixed HHVM tests compatibility

SilverFire committed:
Fixed HHVM tests compatibility. Try 2

SilverFire committed:
Fixed HHVM tests compatibility. Try 3

SilverFire committed:
Skip FormatConverterTest::testPHPDefaultFormat on HHVM.

arogachev committed:
Rename `yii\base\InvalidParamException` to `yii\base\InvalidArgumentException` (#13082)

* Closes #13080: Rename `yii\base\InvalidParamException` to `yii\base\InvalidArgumentException`

* Fixed letter case

* Forgot to rename file :sob:

* Reordered changes in 2.1.0 changelog [skip ci]

* Corrected CHANGELOG to show better diff [skip ci]

arogachev committed:
Fixed typo in CHANGELOG (#13164)

masterklavi committed:
Changed return value in MemCache::setValues()

samdark committed:
Merge branch 'master' into 2.1

# Conflicts:
#	docs/guide/output-client-scripts.md
#	docs/guide/runtime-routing.md
#	framework/BaseYii.php
#	framework/db/oci/Schema.php
#	framework/i18n/Formatter.php
#	framework/log/Logger.php
#	framework/validators/UniqueValidator.php
#	tests/framework/ar/ActiveRecordTestTrait.php
#	tests/framework/widgets/ListViewTest.php

samdark committed:
Fixed className() usage in tests

samdark committed:
Fixed merge bug

samdark committed:
Adjusted widget tests

samdark committed:
Moved masked input field widget into separate extension https://github.com/yiisoft/yii2-maskedinput

aquy committed:
Fixes #13369: Added ability to render current `yii\widgets\LinkPager` page disabled

samdark committed:
Merge branch 'master' into 2.1

# Conflicts:
#	docs/guide/input-validation.md
#	framework/caching/DbDependency.php
#	framework/validators/UniqueValidator.php
#	tests/framework/validators/UniqueValidatorTest.php

Commit:
Fixes #10346: Fixed "DOMException: Invalid Character Error" in `yii\web\XmlResponseFormatter::buildXml()`

samdark committed:
Merge branch 'master' into 2.1

# Conflicts:
#	composer.json
#	docs/guide-ja/output-client-scripts.md
#	framework/caching/XCache.php
#	tests/framework/validators/UniqueValidatorTest.php
#	tests/framework/web/UrlManagerTest.php

Commit:
Fixes #11719: Fixed `yii\db\Connection::$enableQueryCache` caused infinite loop when the same connection was used for `yii\caching\DbCache`

Kolyunya committed:
Fixes #13560: Refactored `\yii\widgets\FragmentCache::getCachedContent()`, added tests

samdark committed:
Merge branch 'master' into 2.1

# Conflicts:
#	framework/i18n/Formatter.php

klimov-paul committed:
Fixed compatibility with PHPUnit 6.x

klimov-paul committed:
Methods `addHeader()`, `setHeader()`, `getHeader()`, `setHeaders()` have been added to `yii\i18n\MessageInterface` allowing setup of custom message headers

klimov-paul committed:
unit test for `mail` fix

klimov-paul committed:
`className()` method usage removed

klimov-paul committed:
fix `db` unit tests

samdark committed:
Improved intl tests to tolerate ICU data differences

- Added new assertion.
- Adjusted tests not to rely on intl version allowing multiple valid variants for Russian and Ukrainian currencies and transliteration.

klimov-paul committed:
unit test `CommandTest` fix

mikehaertl committed:
Fixes #13689: Fixed handling of errors in closures

yyxx9988 committed:
Fixes #13820: Add new HTTP status code 451

Kolyunya committed:
Fixes #13823: Refactored migrations template

SilverFire committed:
Enhanced PHPUnit 6 compatibility

Kolyunya committed:
Fixes #13822: Fixed `yii\web\User::loginRequired()` to throw an `UnauthorizedHttpException` instead of a `ForbiddenHttpException`

samdark committed:
Added UPGRADE note for #13829

Kolyunya committed:
Fixes #13827 Generate "safe" methods uncommented by default

Kolyunya committed:
Fixes #13813: Fixed PHP 7 compatibiltiy by adding support for passing instances of `Error` to the `yii\web\Response::setStatusCodeByException()`

samdark committed:
Removed change from changelog since it's for code which wasn't yet released [skip ci]

samdark committed:
Merge branch 'master' into 2.1

# Conflicts:
#	framework/db/mssql/QueryBuilder.php
#	framework/helpers/BaseHtml.php
#	framework/rbac/DbManager.php
#	framework/rbac/PhpManager.php
#	framework/validators/UniqueValidator.php
#	framework/web/Response.php
#	tests/framework/db/ActiveQueryTest.php
#	tests/framework/db/ActiveRecordTest.php
#	tests/framework/filters/AccessRuleTest.php
#	tests/framework/rbac/ManagerTestCase.php
#	tests/framework/test/ActiveFixtureTest.php
#	tests/framework/web/ErrorActionTest.php

Kolyunya committed:
Fix tests of a web response

SilverFire committed:
Merge pull request #13834 from Kolyunya/13813-fix-tests-of-a-web-response

Fix tests of a web response

takoe committed:
Fixed Russian translation of core-code-style.md (#13832) [skip ci]

MysteryDragon committed:
Fixes #13821: fixed docs and added more tests for Url helper

kalessil committed:
Fixes #13845: `mt_rand()` is not used instead of `rand()` in `yii\captcha\CaptchaAction` + minor code improvements

samdark committed:
Fixed typo [skip ci]

samdark committed:
Fixes #13864: Added info about configuring sub-rules of yii\rest\UrlRule [skip ci]

SamMousa committed:
Fixes #13376: Data provider now automatically sets an ID so there is no need to set it manually in case multiple data providers are used with pagination

SamMousa committed:
Fixes #13837: Refactored masking of CSRF tokens

samdark committed:
Reverted #13822, clarified exception descriptions

phongtt3 committed:
improve vi lang for intro-yii.md (#13886) [skip ci]

IgorCode committed:
Fixed Serbian translation (#13882) [skip ci]

Fixed Serbian translation issue with totalCount appearing twice in GridView summary

phongtt3 committed:
Guide vi improvement (#13894) [skip ci]

* improve vi lang for intro-yii.md

* update Readme & start installation more readable for Vietnamese

SamMousa committed:
Fixes #13883: `\yii\data\SqlDataProvider` now provides automatic fallback for the case when `totalCount` is not specified

samdark committed:
Removed unnecessary else

Kolyunya committed:
Fixes #13770: Added support for `yii\widgets\Menu` item classes definition in the form of an array

samdark committed:
Fixes #13831: Added more info about transactions in migrations to phpdoc [skip ci]

samdark committed:
Eliminated else branches in yii\web\Request

Kolyunya committed:
Fixes #13901: Fixed passing unused parameter to `formatMessage()` call in `\yii\validators\IpValidator`

amin3mej committed:
Fixed persian translation (#13906) [skip ci]

Commit:
Internals PL updates (#13907) [skip ci]

Kolyunya committed:
Fix phpDocumentor annotations (#13905) [skip ci]

Commit:
PL guide updated (#13909) [skip ci]

w3roman committed:
Incorrect use case of Datepicker widget (#13903) [skip ci]

Commit:
Code style headers fix (#13908) [skip ci]

s1lver committed:
Fixed russian translate

SilverFire committed:
Merge pull request #13912 from s1lver/rest_rus_fix

Fixed russian translate

cebe committed:
reverted wrong phpdoc change from #13905

Commit:
Guide PL updates (#13914) [skip ci]

gonimar committed:
Updated Tajik message translations [skip ci] (#13917)

cebe committed:
cleanup some phpdoc

cebe committed:
added more assertations for security masking tests

cebe committed:
remove upgrade note for #13822 / #13829

samdark committed:
Fixed misleading docs about encoded URIs [skip ci]

girmate committed:
Update input-validation.md (#13942) [skip ci]

samdark committed:
Fixes #13945: Removed Courier New from error page fonts list since it looks bad on Linux (#13947)

Commit:
Fixed phpdoc broken links to PHP manual pages [skip ci]

arogachev committed:
Fixes #10675: Added docs for disabling CSRF validation in standalone actions

cebe committed:
updated docs for base64UrlEncode()

as discussed in
https://github.com/yiisoft/yii2/pull/13411#issuecomment-285695112

Kolyunya committed:
Fix `\yii\db\QueryTrait` type hints (#13956) [skip ci]

Kolyunya committed:
Fix `\yii\db\QueryInterface` type hints (#13959) [skip ci]

Faryshta committed:
Added missing parts of disabling CSRF validation doc [skip ci] (#13966)

s1lver committed:
Fixed wording in Russian translation (#13970) [skip ci]

vsguts committed:
Fixes #13961: RBAC Rules: PostgreSQL: PHP Warning "unserialize() expects parameter 1 to be string, resource given" was fixed

klimov-paul committed:
compatibility for PHPUnit adjusted

klimov-paul committed:
compatibility with PHPUnit 6.x added

klimov-paul committed:
unit test fix

SilverFire committed:
`yii\caching\Cache::getOrSet()` now supports both `Closure` and `callable`

Closes #13981

klimov-paul committed:
compatibility with PHPUnit 6.x fix

klimov-paul committed:
garbage collection added to unit tests

klimov-paul committed:
compatibility with PHPUnit 6.x fix

klimov-paul committed:
fixed tests without assettions

klimov-paul committed:
compatibility with PHPUnit 6.x fix

klimov-paul committed:
compatibility with PHPUnit 6.x fix

klimov-paul committed:
memory usage at unit test reduced

cebe committed:
update phpdoc annotation in BaseActiveRecord

https://github.com/yiisoft/yii2-gii/pull/267#issuecomment-293958830

cebe committed:
added upgrade note about yii\cache\Cache::getOrSet()

cebe committed:
Revert "update phpdoc annotation in BaseActiveRecord"

This reverts commit 0f78008bc71af6aca45152452f9f5017e7361e6a.

https://github.com/yiisoft/yii2-gii/pull/267#issuecomment-293965278

paulzi committed:
Fixes #13911: Significantly enhanced MSSQL schema reading performance

acorncom committed:
Tweaked verbiage slightly [ci skip]

SilverFire committed:
Merge pull request #13992 from acorncom/patch-2

Tweaked verbiage slightly [ci skip]

micro-maureen committed:
#13975: make documentation of current behaviour more clear [skip ci]

charlesqwu committed:
docs/guide/input-multiple-models: added a note on avoiding validating the same input data with the same rules more than once (#13995) [skip ci]

sergeymakinen committed:
Fixes #13807: Fixed `yii\db\QueryBuilder` to inherit subquery params when building a `INSERT INTO ... SELECT` query

MysteryDragon committed:
Tiny documentation fix [skip ci] (#13998)

enigmatix committed:
Added note to findOne and findAll docs (#14002) [skip ci]

Added note to findOne and findAll docs to clearly state that complex conditions are not supported.

mikk150 committed:
[IpValidator] disable ipv6 checks (#13984)

* disable ipv6 checks

* remove empty line

* updated CHANGELOG.md

SilverFire committed:
Added AF_INET6 to requirements

SilverFire committed:
Enhanced requirements memo for IPv6

cebe committed:
update AR docs

samdark committed:
Update composer fxp plugin version in docs and travis scripts

vladis84 committed:
Fixes #13963: Added tests for yii\behaviors\TimestampBehavior

samdark committed:
Skipped segfaulting test when running on Travis and corresponding PHP versions

klimov-paul committed:
enforce `backupGlobals` enabled for PHPUnit

klimov-paul committed:
Fix `TimestampBehaviorTest` to be compatible with PHPUnit 6

SilverFire committed:
Switched to asset-packagist

SilverFire committed:
Enable xdebug for coverage on 7.1 insted of 5.6 as it should work faster

SilverFire committed:
Switched ocular to use PHP 7.1 execution results

SilverFire committed:
Merge pull request #14017 from SilverFire/test-asset-packagist

Switched tests to asset-packagist

nurbeknurjanov committed:
Added note clarifying that load function clears fixture data [skip ci]

Kolyunya committed:
Fixes #13728: Fixed the bug when `yii\behaviors\SluggableBehavior` wasn't preserving immutable slug values

samdark committed:
Fixes #13362: Fixed return value of

Insolita committed:
Fixes #14012: `yii\db\pgsql\Schema::findViewNames()` was skipping materialized views

samdark committed:
Created issue template

samdark committed:
Wrong repo [skip ci]

Kolyunya committed:
Fixes #14033: Fixed `yii\filters\AccessRule::matchIp()` erroring in case IP is not defined under HHVM

samdark committed:
Fixes #13694: yii\widgets\Pjax now sends X-Pjax-Url header with response to fix redirect

samdark committed:
Fixed PHP manual link language [skip ci]

samdark committed:
Merged master into 2.1

samdark committed:
Replaced className() with class

germanow committed:
Add setup for Apache

Describes the settings for Apache.

vladis84 committed:
yii\filters\RateLimiter refactored (#13994)

Refactored yii\filters\RateLimiter and added tests

SilverFire committed:
Fixed spelling and typos

SilverFire committed:
Merge pull request #14044 from germanow/germanow-patch-1

Add setup for Apache

depesr committed:
Updated slovak localization

Fixed missing translation for '{attribute} must be an integer.' string

SilverFire committed:
Merge pull request #14045 from depesr/patch-2

Updated slovak localization

SamMousa committed:
Fixed ignored params when using count in `SqlDataProvider`

SilverFire committed:
Merge pull request #14047 from SamMousa/fix-14046

Fixed ignored params when using count in `SqlDataProvider`

cebe committed:
improve session PHPdoc

leandrogehlen committed:
Added `yii\data\Sort::parseSortParams`

allowing customize other params request formats in descendant class.

close #13170

klimov-paul committed:
Fixes #13087: Fixed getting active validators for safe attribute

samdark committed:
Adjusted code style for less conditions branching

samdark committed:
Fixes #13226: `yii cache` command now warns about the fact that it's not able to flush APC cache from console

Commit:
Fixes #13848: `yii\di\Instance::ensure()` wasn't throwing an exception when `$type` is specified and `$reference` object isn't instance of `$type`

cebe committed:
Helper independent (#14050)

* Made sure most helpers are independent of Yii::$app

* Made sure most validators are independent of Yii::$app

samdark committed:
Removed redundant use

samdark committed:
Removed redundant else

kernight committed:
Update runtime-sessions-cookies.md

修改错别字

kernight committed:
Update runtime-responses.md

修改错别字

kernight committed:
Merge branch 'doc' into master

kernight committed:
Merge branch '2.1' into master
